.github/workflows/stale.yml

```
@@ -0,0 +1,25 @@
+name: 'Close stale issues and PRs'
```

```suggestion
name: 'Flag and then close stale PRs'
```
is more appropriate?